collections/ui/src/index.js

```diff
@@ -49,6 +49,8 @@ export { Node } from '@dhis2-ui/node'
 export { NoticeBox } from '@dhis2-ui/notice-box'
 export {
     OrganisationUnitTree,
+    RootError,
+    RootLoading,
```

Comment:
Should we prefix these exports?

```suggestion
    RootError as OrganisationUnitTreeRootError,
    RootLoading as OrganisationUnitTreeRootLoading,
```

Reply:
Good idea